templates/repo/commit_statuses.tmpl

```diff
@@ -1,10 +1,10 @@
 {{if .Statuses}}
 	{{if and (eq (len .Statuses) 1) .Status.TargetURL}}
-		<a class="tw-align-middle {{.AdditionalClasses}} tw-no-underline" data-tippy="commit-statuses" href="{{.Status.TargetURL}}">
+		<a class="commit-statuses tw-align-middle {{.AdditionalClasses}} tw-no-underline" data-tippy="commit-statuses" href="{{.Status.TargetURL}}">
```

Comment:
Thank you, that's a good fix.

ps: I can see the misalignment in your screenshot. Maybe it could be like this:


```suggestion
		<a class="commit-statuses flex-text-block muted {{.AdditionalClasses}}" data-tippy="commit-statuses" href="{{.Status.TargetURL}}">
```

?? 

(haven't tested)

Reply:
done but are we sure to use muted class since we dont have text there?

Reply:
Not quite sure. `tw-no-underline` vs `muted`, which one is right? 🤔

Reply:
no need to both i think because we dont have text there :)